install_config/persistent_storage/persistent_storage_local.adoc

```diff
@@ -0,0 +1,80 @@
+[[install-config-persistent-storage-persistent-storage-local]]
+= Persistent Storage Using Local Volume
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+{product-title} clusters can be provisioned with xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+storage] using local volumes. Local persistent volumes allows users to access local storage device such as a disk, partition or directory through the standard PVC interface. An external static provisioner and a related bootstrapper are available to help simplify local storage management after the local volumes are configured.
+
+Local volumes can be used without manually scheduling pods to nodes, as the system is aware of the volume’s node constraints. However, local volumes are still subject to the availability of the underlying node and are not suitable for all applications.
+
+[NOTE]
+====
+. Local volume is an alpha feature and may change in a future release of {product-title}.
+. The local storage provisioner is different from most provisioners and does not support dynamic provisioning.
+. Local storage provisioner requires that administrators preconfigure the local volumes on each node and mount them under discovery directories. The provisioner then manages the volumes by creating and cleaning up PersistentVolumes for each volume.
+====
+
+[WARNING]
+====
+Local volumes can only be used as a statically created Persistent Volume.
+====
+
+[[local-volume-usage]]
+== Using a Local Volume
+//add intro
+[[local-volume-starting-cluster-with-local-disks]]
+=== Starting a Cluster with Local Disks
+[[local-volume-baremetal-environments]]
+==== Baremetal Environments
+. Partition and format the disks on each node (according to the application's requirements).
+. Mount all the filesystems under one directory per `StorageClass`. The directories are specified in a *_configmap_*, see below. By default, the discovery directory is *_/mnt/disks_* and storage class is `local-storage`.
```

Comment:
s/Mount all the filesystems under one directory per `StorageClass`. The directories are specified/Mount all the partitions under the discovery directory per `StorageClass`. The directory is specified./